<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <listeners>
    <listener class-name="ke.TestComponent.Listeners"/>
  </listeners>
  <test thread-count="5" parallel="tests" name="ErrorValidations(failed)">
    <classes>
      <class name="ke.tests.ErrorValidations">
        <methods>
          <include name="productErrorValidation"/>
          <include name="quit"/>
          <include name="launchApplication"/>
        </methods>
      </class> <!-- ke.tests.ErrorValidations -->
    </classes>
  </test> <!-- ErrorValidations(failed) -->
  <test thread-count="5" parallel="tests" name="SubmitOrder(failed)">
    <classes>
      <class name="ke.tests.StandloneTest">
        <methods>
          <include name="submitOrder" invocation-numbers="0"/>
          <include name="launchApplication"/>
          <include name="quit"/>
        </methods>
      </class> <!-- ke.tests.StandloneTest -->
    </classes>
  </test> <!-- SubmitOrder(failed) -->
</suite> <!-- Failed suite [Suite] -->
